src/api/User/types.ts

```diff
@@ -8,6 +8,8 @@ import {GenericObject, IDataSourceParams, PersistedItem} from '../commonTypes';
 import {AccessRight, Action, AnyAction} from '../AccessRight';
 import {IGuestPreferencesConfig} from '../Config';
 
+export type DELETE_USER = 'Deleted user';
```

Comment:
Since this is a type, let's follow the convention and rename the variable to `DeletedUser`